equinix/resource_metal_virtual_circuit_acc_test.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/packethost/packngo"
 )
 
+const (
```

Comment:
I think this might've leaked in from another change?

Reply:
No, this constant was being reused in virtual circuit as well. Hence had to copy that separately